### PR TITLE
MISUV-3697 Homepage changes for payment history refunds title

### PR DESCRIPTION
**Pull request by vinay1yv**
…ame change

### CODE REVIEW

File: app/views/Home.scala.html [hunk 1]
```diff
@@ -246,31 +247,54 @@ <h1 class="govuk-heading-xl @if(user.saUtr.isDefined) { margin-bottom-zero }">
...
-        } else {
-            @link(
-                link = creditAndRefundsControllerURL,
-                messageKey = "home.credAndRefund_howToClaim.view"
-            )
-        }
-    }
-}
+    @if(paymentHistoryEnabled) {
+             @home_tile("payment-history-tile", messages("home.paymentHistoryRefund.heading")) {
```
Comment (joesheridan) [inline app/views/Home.scala.html:251]: please fix all formatting here - should be 4 spaces for a tab

File: test/views/HomePageViewSpec.scala [hunk 2]
```diff
@@ -245,14 +246,23 @@ class HomePageViewSpec extends TestSupport {
 
     "have a payment history tile" which {
       "has a heading" in new Setup {
-        getElementById("payment-history-tile").map(_.select("h2").text) shouldBe Some(messages("home.paymentHistory.heading"))
+        getElementById("payment-history-tile").map(_.select("h2").text) shouldBe Some(messages("home.paymentHistoryRefund.heading"))
```
Comment (joesheridan) [inline test/views/HomePageViewSpec.scala:249]: there should be 2 tests: one for when the feature switch is enabled, and one where it is disabled and we should see the value changing..

File: test/views/agent/HomePageViewSpec.scala [hunk 3]
```diff
@@ -89,7 +89,9 @@ class HomePageViewSpec extends TestSupport with FeatureSwitching with ViewSpec {
       dunningLockExists,
       currentTaxYear,
       isAgent,
-      creditAndRefundEnabled = false
+      creditAndRefundEnabled = false,
+      paymentHistoryEnabled = isEnabled(PaymentHistoryRefunds)
```
Comment (joesheridan) [inline test/views/agent/HomePageViewSpec.scala:93]: there should be tests with paymetnhistoryrefunds enabled and disabled, same as above

File: test/views/HomePageViewSpec.scala [hunk 4]
```diff
@@ -245,14 +246,23 @@ class HomePageViewSpec extends TestSupport {
...
 
       "has a link to the payment and refund history page" in new Setup {
         val link: Option[Element] = getElementById("payment-history-tile").map(_.select("a").first)
         link.map(_.attr("href")) shouldBe Some(controllers.routes.PaymentHistoryController.show().url)
-        link.map(_.text) shouldBe Some(messages("home.paymentHistory.view"))
+        link.map(_.text) shouldBe Some(messages("home.paymentHistoryRefund.view"))
       }
+
+
+      "has an link to the 'How to claim a refund' for not migrated user" in new Setup(user = testMtdItUserNotMigrated()) {
```
Comment (joesheridan) [inline test/views/HomePageViewSpec.scala:259]: was this part of your ticket? I thought Pavel was doing something about this?

File: app/views/Home.scala.html (hunk 1 above)
Comment (vinay1yv) [reply app/views/Home.scala.html]: Hi,Fixed the formatting issues.

File: test/views/HomePageViewSpec.scala (hunk 4 above)
Comment (vinay1yv) [reply test/views/HomePageViewSpec.scala:270]: There was comment around the test saying it should be fixed with MISUV-3609,and it was failing as it has step to test page heading.So changed it to new heading and it passed.

**Review by joesheridan — APPROVED** [review]

**Pull request merged**